test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -285,6 +285,9 @@ public void SdcaMulticlass()
             Assert.NotNull(pred);
             VBuffer<float>[] weights = default;
             pred.GetWeights(ref weights, out int n);
+
+            var biases = pred.GetBiases();
```

Comment:
>GetBiases [](start = 30, length = 9)

Is where are case in which user want's only weights or only biases? If no, I would suggest to rename GetWeights to GetWeightsAndBiases

Reply:
let's leave them decoupled; it's usually more flexible. 

---
In reply to: [224521992](https://github.com/dotnet/machinelearning/pull/1224#discussion_r224521992) [](ancestors = 224521992)